odoo/addons/base/tests/test_db_cursor.py

```diff
@@ -50,20 +50,20 @@ def test_multiple_close_call_cursor(self):
 
     def test_transaction_isolation_cursor(self):
         with registry().cursor() as cr:
-            self.assertEqual(cr.connection.isolation_level, ISOLATION_LEVEL_REPEATABLE_READ)
```

Comment:
There is a few changes like this. This was correct and was using the public-API - why change?